extension/src/inscription-view/open-page.ts

```diff
@@ -0,0 +1,43 @@
+import * as vscode from 'vscode';
+import * as fs from 'fs';
+import * as path from 'path';
+import { InscriptionActionArgs } from '@axonivy/inscription-protocol';
+import { InscriptionActionHandler } from '../inscription-action-handler';
+
+export class OpenPageActionHandler implements InscriptionActionHandler {
+  actionId = 'openPage' as const;
+  async handle(actionArgs: InscriptionActionArgs): Promise<void> {
+    if (actionArgs.actionId === this.actionId) {
+      const path = actionArgs.payload.toString();
+      if (isUrl(path)) {
+        openUrlInBrowser(path);
```

Comment:
ah cool thanks, but I think I'll use the external browser for now. Unless there is a good reason to use the integrated one, otherwise it would be quickly changed 👍

Reply:
if you plan to add a test, what would be cool ;) it would be easier with the integrated one

Reply:
Ah okay, then I will use the built-in one. However, I'll do this in another PR where I'll also try to write appropriate playwright tests for it or can i also just write Vitest-Unittests in this repo?

Reply:
up to now there are only playwright tests